Dockerfile

```diff
@@ -1,76 +0,0 @@
-# FROM java:8
-FROM alpine:3.7
-
-ENV SONAR_VERSION=6.1 \
-    SONARQUBE_HOME=/opt/sonarqube \
-    SONAR_RUNNER_VERSION=2.4 \
-    SONAR_RUNNER_HOME=/opt/sonar-runner \
-    # Database configuration
-    # Defaults to using H2
-    SONARQUBE_JDBC_USERNAME=sonar \
-    SONARQUBE_JDBC_PASSWORD=sonar \
-    SONARQUBE_JDBC_URL=
-
-ENV JAVA_HOME="/usr/lib/jvm/java-1.8-openjdk"
```

Comment:
Looks like Sonar 7.8 is compatible with java 11 if we want to try it out.

Reply:
I'm good with upgrading it at a later time. Pulled most of this from existing dockerfile.